packages/checkout/widgets-lib/src/widgets/connect/ConnectWidgetRoot.tsx

```diff
@@ -24,15 +27,18 @@ export class Connect extends Base<WidgetType.CONNECT> {
 
   protected render() {
     if (!this.reactRoot) return;
+    const { t } = i18n;
 
     this.reactRoot.render(
       <React.StrictMode>
         <CustomAnalyticsProvider checkout={this.checkout}>
           <ThemeProvider id="connect-container" config={this.strongConfig()}>
-            <ConnectWidget
-              config={this.strongConfig()}
-              checkout={this.checkout}
-            />
+            <Suspense fallback={<LoadingView loadingText={t('views.LOADING_VIEW.text')} />}>
```

Comment:
Just wondering if this adds another loading view flicker while we wait for the bundle to load. I'll pull down and test

Reply:
Locally for me when I tested it looked like the spinner reset its location when it moved to the next widget's loading screen but I didn't see a flicker when changing from the connect -> the widget other than that. But yep please pull down and try it out 👍 I did this on a slower network. Before I added the fallback there was a while where the widget was removed from the screen before the actual widget loaded (like when you are going from connect -> swap for example)